### DISCUSSION TITLE
Official vs custom build differences?

### DISCUSSION THREAD

**Discussion by lin-ycv** [KlipperScreen/KlipperScreen#1214] [Q&A]
I'm trying to do a fresh install on a Flsun speeder pad. I'm wondering what the difference is between the using `kiauh` to install klipper screen vs cloning `https://github.com/Guilouz/KlipperScreen-Flsun-Speeder-Pad`
is it just the macros/setting listed in the [notes section](https://github.com/Guilouz/KlipperScreen-Flsun-Speeder-Pad?tab=readme-ov-file#notes)? or are there critical differences between official vs custom build?

the long story short is: if I install the official build with `kiauh`, would there be any imcompatibility issues?
